frontend/src/views/Main.vue

```diff
@@ -55,58 +51,25 @@ export default {
       });
     },
     postPost() {
-      const path = 'http://localhost/api/post';
-      const data = { username: this.userInfo.username, text: this.tweet, spotify_data: null}
+      const path = 'http://localhost/api/social/post';
+      const data = { username: this.currentUser.username, text: this.tweet, spotify_data: null}
       axios.post(path, data, {
         headers: authHeader()
       })
+      .then(()=> {
+        this.getPosts();
+      })
       .catch(error => {
         console.error("Could not POST user post", error);
       })
-      this.getPosts();
+      
     },
     updateValue(e) {
       this.$emit('input', e.target.value);
     },
-    getUserInfo() {
-      const path = 'http://localhost/api/auth/get_user_info';
-      var userInfo = this.currentUser ? this.currentUser : ''
-      axios.get(path, {
-        params: {
-          access_token: userInfo.access_token ? userInfo.access_token : ''
-        },
-        headers: authHeader()
-      })
-      .then((res) => {
-        this.userInfo = res.data.userInfo ? res.data.userInfo : res.data.status
-      })
-      .catch((error) => {
-        console.error(error);
-        this.userInfo = 'User not logged in'
-      });
-    },
   },
   created() {
-    this.getUserInfo();
     this.getPosts();
   },
 }
-</script>
-
-<style lang="scss">
-@import '~normalize.css';
-@import '~reset-css/sass/_reset.scss';
-* {
-  box-sizing: border-box;
-}
-html body {
-  color: #444;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif;
-}
-.App {
-  &__messageBox {
-    padding: 1em;
-    background-color: #e5f6f8;
-  }
-}
```

Comment:
This stuff either wasn't doing anything/the color broke the theming by not allowing text-color to change when switching between dark/light modes